demos/coreHTTP/http_demo_s3_download_multithreaded.c

```diff
@@ -909,18 +880,22 @@ void prvTearDown( TaskHandle_t xHandle,
  * This example, using a pre-signed URL, resolves a S3 domain, establishes a TCP
  * connection, validates the server's certificate using the root CA certificate
  * defined in the config header, then finally performs a TLS handshake with the
- * HTTP server so that all communication is encrypted.
- *
- * Afterwards, an independent HTTP task is started, to read requests from the
- * request queue and execute them using the HTTP Client library API, and write
- * the corresponding responses to the response queue. The main task sends
- * requests to the request queue, which are used to download the S3 file in
- * chunks using range requests. While doing so, the main task reads responses
- * from the response queue continuously until the entire file is received. If
- * any request fails, an error code is returned.
+ * HTTP server so that all communication is encrypted. The AWS S3 service validates
```

Comment:
See previous comment about server certificate validation being part of the TLS handshake, the language on lines 880-883 can be made more clear.

Reply:
Fixed the language